Taskfile.yml

```diff
@@ -156,12 +156,10 @@ tasks:
   test-ui:
     desc: Run UI tests {{.XVFB}}
     aliases: [ui]
-    env:
-      MOCHA_GREP: Test One Click Trial feature
-      MOCHA_INVERT: true
     cmds:
       - yarn webpack-dev
       - "{{.VIRTUAL_ENV}}/bin/python3 --version"
+      - bash -c 'rm -rf ./out/test-resources ./out/ext'
```

Comment:
there is no need to run bash here, it would work without too.

Reply:
Thanks for the information!